samples/AnomalyDetector/tests/Generated/Tests/AnomalyDetectorClientTests.cs

```diff
@@ -0,0 +1,740 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using AnomalyDetector;
+using AnomalyDetector.Models;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace AnomalyDetector.Tests
+{
+    public partial class AnomalyDetectorClientTests : AnomalyDetectorTestBase
+    {
+        public AnomalyDetectorClientTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [Test]
+        [Ignore("Only validating compilation of test scaffoldings")]
```

Comment:
Does customers see the same `Ignore`?

Reply:
Yes. We must have this Ignore. Otherwise our pipeline in this repo will try to run the test and fail

Reply:
Then it is better to be like `[Ignore("Remove this line to start writing tests")]`?

Reply:
I changed the message to `Please remove the Ignore attribute to let the test method run`